bentoml/marshal/utils.py

```diff
@@ -2,6 +2,10 @@
 from functools import lru_cache
 from typing import NamedTuple, Iterable
 
+from bentoml import config as bentoml_config
+
+BATCH_REQUEST_HEADER = bentoml_config("apiserver").get("batch_request_header")
```

Comment:
is there any use case for the user to customize this config? maybe we should just make it a constant?